pkg/verify/sct.go

```diff
@@ -52,16 +52,21 @@ func VerifySignedCertificateTimestamp(leafCert *x509.Certificate, threshold int,
 		}
 
 		for _, fulcioCa := range fulcioCerts {
+			fulcioChain := make([]*ctx509.Certificate, len(certChain))
```

Comment:
nit: Could we rename line 41 to be `leafCTCert` or similar rather than `certChain`, helps readability to see that it should only be a single certificate.
Also can assert that it parsed only a single certificate with a length check

Reply:
++ to clearer names!

Since `leafCert` is a `*x509.Certificate`, I don't think there's a case where we'd end up with more than one certificate? I'm going to leave that off for now, but happy to add if I misunderstand.